src/Microsoft.AspNetCore.Http.Connections.Common/NegotiateProtocol.cs

```diff
@@ -18,6 +18,8 @@ public static class NegotiateProtocol
         private const string AvailableTransportsPropertyName = "availableTransports";
         private const string TransportPropertyName = "transport";
         private const string TransferFormatsPropertyName = "transferFormats";
+        // Used to detect ASP.NET SignalR Server connection attempt
+        private const string TryWebSocketsPropertyName = "TryWebSockets";
```

Comment:
Can we use `ProtocolVersion` instead? It's still unique to ASP.NET SignalR but it's a little more of an obvious property to use when detecting.